Automattic/wp-calypso
# Marketplace: Create a component to display reviews

Related to https://github.com/Automattic/dotcom-forge/issues/4562

## Proposed Changes

Create a component that receives product slug and product type and display all the reviews for this product.

| Desktop  | Mobile |
| ------------- | ------------- |
|![CleanShot 2023-11-29 at 14 04 26@2x](https://github.com/Automattic/wp-calypso/assets/5039531/d7519e5d-f354-4425-b858-60607f56b4c0)|![CleanShot 2023-11-29 at 14 05 55@2x](https://github.com/Automattic/wp-calypso/assets/5039531/719127c8-2235-4325-951d-69ce29dcbd76)|


### Out of scope
* Pagination will be handled  as a follow-up
* The renaming of the types from plugin to product are handled on https://github.com/Automattic/wp-calypso/pull/84480/

## Testing Instructions
* Add to the `plugin-details.jsx` file
* On the import area 
```ts
import { MarketplaceReviewsList } from '../marketplace/components/reviews-list';
```

* On line `454`
```ts
<MarketplaceReviewsList slug={ fullPlugin.slug } productType="plugin" />
```
* Go to a plugin page with reviews. Ex: `plugins/woocommerce-bookings`
* Go to the bottom of the page and check if the reviews are displayed there
* Also test on the mobile view
* Also check plugins without a reviews. Ex: `/plugins/wordpress-seo-premium`



## Pre-merge Checklist
